.vscode/settings.json

```diff
@@ -1,5 +1,7 @@
 {
-  "editor.rulers": [80],
+  "editor.rulers": [
+    80
+  ],
```

Comment:
I'm curious why the change is needed. `npm run lint` does not check this file, right?

Reply:
@raymondfeng nope, I my vscode breaks it into multiple lines automatically when I save the file...

See discussion in https://github.com/strongloop/loopback-next/pull/2690/files#diff-5b46cfda0ddaed371f52aa24ea2b0b2a

Reply:
I tried to save this file and VS code doesn't reformat for me. Just FYI.

Reply:
ok seems other people don't have the problem. I can close the PR.